ratify.yaml

```diff
@@ -0,0 +1,157 @@
+package:
+  name: ratify
+  version: "1.4.0"
+  epoch: 0
+  description: Artifact Ratification Framework (CNCF Sandbox)
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: f3f6229dee1051e4beb580ec17ad68b7a45f23af
+      repository: https://github.com/notaryproject/ratify
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.38.0
+        github.com/go-jose/go-jose/v3@v3.0.4
+        github.com/go-jose/go-jose/v4@v4.0.5
+        github.com/golang-jwt/jwt/v4@v4.5.2
+        github.com/golang-jwt/jwt/v5@v5.2.2
+        github.com/cloudflare/circl@v1.6.1
+        github.com/open-policy-agent/opa@v1.4.0
+        github.com/go-viper/mapstructure/v2@v2.3.0
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X github.com/ratify-project/ratify/internal/version.GitCommitHash=$(git rev-parse HEAD)
+        -X github.com/ratify-project/ratify/internal/version.GitTag=${{package.version}}
```

Comment:
should we add: -X github.com/ratify-project/ratify/internal/version.GitTreeState=$(git diff --quiet && echo clean || echo dirty) as done here: https://github.com/notaryproject/ratify/blob/main/Makefile#L25C24-L25C54

Reply:
Does goreleaser add that header already? As we're not using goreleaser we might need to add it. Can you check the upstream binary vs ours and see if we're missing any ldflags